src/dataflow/src/source/mod.rs

```diff
@@ -1766,6 +1795,8 @@ impl SourceReaderPersistence {
                 .or_default() += diff;
         }
 
+        log::debug!("Lowest restored bindings ts: {}", lowest_bindings_ts);
```

Comment:
couldn't this binding actually correspond to a zero diff?

Reply:
It could, I added this debug message to see what's happening with compaction, and if the lowest message we get is actually beyond the seal upper, which would be bad. I think we won't keep all the logging changes, that's why I put them out in separate commits.